docs/migration/mui_migration_1.0.md

```diff
@@ -0,0 +1,15 @@
+# Consumption guide of shared components
```

Comment:
I'd assume this a migration guide not a consumption guide, right? Please adjust the title accordingly.